### ISSUE TITLE
Creation of Database Model

### ISSUE THREAD

**Issue by FendVario1** [amosproj/amos2022ss01-firmware-downloader#11]
## User story
1. As a Developer
2. I want / need a database model
3. So that I know what fields are mandatory and can add additional fields to it, if they should become necessary

## Acceptance criteria
* document with currently planned database model

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by TejeshPala** [comment]
[Database Model.pdf](https://github.com/amosproj/amos2022ss01-firmware-downloader/files/8707941/Database.Model.pdf)

The Database Model describes how the data should be implemented/ used for the Project

**Comment by TejeshPala** [comment]
The necessary documentation and implementation for the Project is modified as per Industry partner discussion today and aligned as required
Moving the issue for Awaiting Review from Dashboard
